packages/components-js/scripts/convertToNextJsVRTPage.ts

```diff
@@ -1,20 +1,30 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import { paramCase } from 'change-case';
 import { convertToReactVRTPage, ReactCharacteristics } from './convertToReactVRTPage';
+import { paramCase } from 'change-case';
 
 const sourceBasePath = path.resolve(__dirname, '../../components-react/src');
 const pollComponentsReadyFilePath = path.resolve(sourceBasePath, 'pollComponentsReady.ts');
-const pollComponentsReadyFileContent = fs.readFileSync(pollComponentsReadyFilePath, 'utf8');
+const pollComponentsReadyFileContent = fs
+  .readFileSync(pollComponentsReadyFilePath, 'utf8')
+  .replace(/export const pollComponentsReady/, 'const pollComponentsReady');
```

Comment:
Why not use a capture group like?
```suggestion
  .replace(/export (const pollComponentsReady)/, '$1');
```